textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/nlpextrator/NlpExtractor.java

```diff
@@ -16,23 +16,25 @@
 import edu.uci.ics.textdb.common.field.Span;
 import edu.uci.ics.textdb.common.utils.Utils;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Properties;
+import java.util.*;
 
 
 /**
- * @author Feng [sam0227] on 4/27/16.
- *         <p>
- *         Wrap the Stanford NLP Named Entity Recognizer as an operator.
- *         This operator would recognize 7 classes: Location, Person, Organization, Money, Percent, Date and Time.
- *         Return the recoginized data as a list of spans that are appended to the original tuple as a field.
- *         <p>
- *         For example: Given tuple with two fields: sentence1, sentence2.
- *         tuple: ["Google is an organization.", "Its headquarter is in Mountain View."]
- *         <p>
- *         Append a list of spans then return:
- *         ["sentence1,0,6,Google, NE_ORGANIZATION", "sentence2,22,25,Mountain View, NE_LOCATION"]
+ * @author Feng
+ * @about Wrap the Stanford NLP as an operator to extractor desire
+ * information (Named Entities, Part of Speech).
+ * This operator could recognize 7 Named Entity classes: Location,
+ * Person, Organization, Money, Percent, Date and Time.
+ * It'll also detect 4 types of Part of Speech: Noun, Verb, Adjective
+ * and Adverb.Return the extracted token as a list of spans and
+ * appends to the original tuple as a new field.
+ * For example: Given tuple with two fields: sentence1, sentence2,
+ * specify to extract all Named Entities.
+ * Source Tuple: ["Google is an organization.", "Its headquarter is in
```

Comment:
"Its headquarter is" -> "Its headquarters are".  It's one thing I learned about English: "headquarter" has to be in plural :-)